package.json

```diff
@@ -6,14 +6,14 @@
     "npm": "6.14.13",
     "node": "14.17.3"
   },
-  "main": "src/main.ts",
+  "main": "src/server.ts",
   "scripts": {
     "build": "gulp",
     "test": "NODE_ENV=production NODE_CONFIG_ENV=testing mocha --exit -r ts-node/register --project tsconfig.json test/**/*.spec.ts",
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "lint:md": "remark .",
-    "start": "node dist/src/main.js",
+    "start": "node dist/src/server.js",
```

Comment:
Can you please rebase - You should not have this change as Akash has already merged PR for main application.

For that matter, there should not be any change in package.json